Carbon/components/FetchMonthEmissions.js

```diff
@@ -0,0 +1,48 @@
+import { API_URL } from "../config/Api";
+
+const TIMEOUT_DURATION = 25000;
+
+/**
+  Fetches month emissions data from the API.
+  @param {string} month - The month for which to fetch emissions data.
+  @param {string} user_id - The ID of the user for whom to fetch emissions data.
+  @returns {Promise} A Promise that resolves with the fetched data, or rejects with an error.
+  @throws {Error} If there was an error fetching the data.
+  @example const data = await FetchMonthEmissions('january', '1234');
+**/
+export const FetchMonthEmissions = async(month, user_id) => {
+  try {
+    // Construct the URL for the API endpoint
+    // const url = `${API_URL}userEmissions/${month}/${user_id}`; // TODO: Connect to the actual API endpoint eventually
+
+    const url = `http://192.168.0.165:3000/api/userEmissions/${month}/${user_id}` // Using the local IPv4 for testing purposes
```

Comment:
After the pull from Eriks's PR we will have to change this into the the Config/Api.js export, so we can just change this. Our program is becoming more and more unified with the server so squashing these things out will make it easier in the future. I think its best to add any routes into that config file. 😄

Reply:
Resolved in [28501cd](https://github.com/CivBuilder/Carbon/pull/88/commits/28501cdee91662a4ead773479a8770cb5159bffe)!